packages/broker/src/ConfigWizard.ts

```diff
@@ -260,11 +262,11 @@ export const createStorageFile = async (config: any, answers: inquirer.Answers):
 
 export const startBrokerConfigWizard = async(): Promise<void> => {
     try {
-        const answers = await inquirer.prompt(prompts)
-        const config = getConfigFromAnswers(answers)
-        if (answers.revealGeneratedPrivateKey) {
-            logger.info(`This is your node\'s private key: ${config.ethereumPrivateKey}`)
-        }
+        let config = { ... CONFIG_TEMPLATE, plugins: { ... CONFIG_TEMPLATE.plugins } }
+        const ethereumAnswers = await inquirer.prompt(privateKeyPrompts)
```

Comment:
Could rename `ethereumAnswers` to `privateKeyAnswers` so that one concept has only one name.